src/Queue/TenantAwareJob.php

```diff
@@ -35,7 +35,9 @@ public function __sleep()
         /** @var Environment $environment */
         $environment = app(Environment::class);
 
-        if ($hostname = $environment->hostname()) {
+        $hostname = $environment->hostname();
+
+        if (isset($hostname) && !isset($this->hostname_id)) {
```

Comment:
I don't really like isset. Please use `if ($hostname && !$this->hostname_id)` instead.

Sorry matter of taste, we can safely assume the value to be null or set anyway.